purity_fb/README.md

```diff
@@ -606,10 +650,7 @@ Class | Method | HTTP request | Description
  - [PolicyMemberResponse](docs/PolicyMemberResponse.md)
  - [PolicyMemberWithRemote](docs/PolicyMemberWithRemote.md)
  - [PolicyMemberWithRemoteResponse](docs/PolicyMemberWithRemoteResponse.md)
- - [PolicyObjects](docs/PolicyObjects.md)
```

Comment:
I _think_ that these files are actually still around and should be included. The code generate just drops them for some reason.
Can you confirm that these .md files exist, and then manually re-add them if they do?

Reply:
yup they exist, manually re-added